appveyor.yml

```diff
@@ -11,13 +11,15 @@ cache:
 # what combinations to test
 environment:
   matrix:
-    - nodejs_version: 0.10
-    - nodejs_version: 0.11
-    - nodejs_version: 0.12
+    - nodejs_version: "8"
+    - nodejs_version: "10"
+    - nodejs_version: "12"
+
+platform:
+  - x64
 
 install:
-  - ps: Install-Product node $env:nodejs_version x64
-  - npm -g install npm@2.12.1
```

Comment:
`npm` version 2... The CLI was installing version 4 and I thought it was very old! :-D

Reply:
@pdcastro I might actually remove that file completely, it seems this doesn't build on appveyor anyways.

Reply:
And yeah, this is not much maintained :/